web/themes/custom/itkdev/itkdev_project_theme/css/project-styles.css

```diff
@@ -88,3 +88,26 @@
 .project-members:last-child .spacing {
     display: none;
 }
+
+.webform-progress .progress-step {
+  color: rgba(0,0,0,.3);
+
+  &.is-complete {
+    color: #047857;
+  }
+
+  &.is-active {
+    color: rgb(0,0,0)
```

Comment:
Could the color be applied with a tailwind class on the markup? Maybe using the primary color.